test/headAlignBugTest.html

```diff
@@ -343,6 +370,58 @@
 var strict = "strict";
 // strict = undefined;
 
+//automated test
+describe('goodTree1: ' + parenthesizeTree(goodTree1), function() {
```

Comment:
- include the expected values in the displayed text for the it() statements
- specify the value of side -- it should be either 'right' or 'left', i think. I'm not sure where it's being defined at present.